checker-qual/src/main/java/org/checkerframework/checker/regex/qual/PolyRegex.java

```diff
@@ -11,9 +11,10 @@
 /**
  * A polymorphic qualifier for the Regex type system.
  *
- * <p>Any method written using {@link PolyRegex} conceptually has two versions: one in which every
- * instance of {@link PolyRegex} has been replaced by {@link Regex}, and one in which every instance
- * of {@link PolyRegex} has been replaced by {@link UnknownRegex}.
+ * <p>Any type annotated by {@link PolyRegex} conceptually can be replaced by the following
+ * qualifiers: {@link Regex}, which can take an integer argument and instantiate to capturing group
+ * argument; {@link UnknownRegex}; {@link PartialRegex}, which can take string argument and
+ * instantiate to different type frames; {@link UnknownRegex}; {@link RegexBottom}.
```

Comment:
The string argument is not a type frame, it's a partial regex.

Reply:
Changed now